### ISSUE TITLE
Add volume property to player

### ISSUE THREAD

**Issue by PassTheMayo** [briantanner/eris-lavalink#20]
There is no volume property within the player. Maybe can you add one?

**Comment by Kaydax** [comment]
What do you mean by "within the player". There is already volume control for a per player basis. All you have to do is get the player for the guild in question and set the volume through it.

**Comment by PassTheMayo** [comment]
Yes, I understand you can adjust the volume, but there's no way to get what the current volume is set to.

**Comment by Kaydax** [comment]
The best way that I do this is to have the volume stored in my database that stores settings per server, and uses that to set the volume and return what the volume is. But i see now what you mean, but the problem is the fact that lavalink only has a way to set but not request the volume.
